tmp.csv

```diff
@@ -0,0 +1,6 @@
+Name     Number   BeamID   E          N          U
```

Comment:
Why did this file make its way into the repo?

Reply:
Woops, my mistake. I've  `git rm`'d it now.